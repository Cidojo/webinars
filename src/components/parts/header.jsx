import React from 'react';
import {Link} from 'react-router-dom';

const Header = () => {
  return (
    <header className='header'>
      <div className='header__inner'>
        <Link to='/'>
          <img className="header__logo" src='../../assets/img/logo.svg' alt='logo' />
        </Link>
      </div>
    </header>
  );
};

export {Header};
