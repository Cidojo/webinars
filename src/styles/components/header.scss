.header {
  background-color: #2A2E32;
}

.header__inner {
  @extend %container;

  display: flex;
  align-items: center;
  justify-content: center;
  padding-top: 16.5px;
  padding-bottom: 16.5px;

  > a {
    display: flex;
  }
}

.header__logo {
  width: 166px;
  height: 17px;
}
