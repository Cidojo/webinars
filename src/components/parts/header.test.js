import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {BrowserRouter} from 'react-router-dom';
import {Header} from './header.jsx';

it(`should render Header component without errors`, () => {
  const tree = renderer.create(<BrowserRouter><Header/></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
