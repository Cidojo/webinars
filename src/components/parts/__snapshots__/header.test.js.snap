// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component without errors 1`] = `
<header
  className="header"
>
  <div
    className="header__inner"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      <img
        alt="logo"
        className="header__logo"
        src="../../assets/img/logo.svg"
      />
    </a>
  </div>
</header>
`;
